default/AI/FreeOrionAI.py

```diff
@@ -159,16 +160,11 @@ def generateOrders():  # pylint: disable=invalid-name
     print "Start turn %s (%s) of game: %s" % (turn, turn_uid, foAIstate.uid)
 
     turn_timer.start("AI planning")
-    universe = fo.getUniverse()
     empire = fo.getEmpire()
-    planet_id = PlanetUtilsAI.get_capital()
     # set the random seed (based on galaxy seed, empire ID and current turn)
     # for game-reload consistency 
     random_seed = str(fo.getGalaxySetupData().seed) + "%03d%05d" % (fo.empireID(), turn)
     random.seed(random_seed)
-    planet = None
-    if planet_id is not None:
-        planet = universe.getPlanet(planet_id)
```

Comment:
Looks like I merge it with issues. Because charting code was moved to handlers too.